snow/consensus/snowman/poll/early_term_no_traversal_test.go

```diff
@@ -372,16 +374,57 @@ func TestEarlyTermTraversal(t *testing.T) {
 	poll.Vote(vdr3, blkID3)
 	require.False(poll.Finished())
 
-	//
-	//        blkID0                       blk0: {0x00, 0x00}
-	//              \3                     blk1: {0x0f, 0x00}
-	//              {0x?f}                 blk2: {0xff, 0xf0}
-	//              1/    \2               blk3: {0x0f, 0xff}
-	//            blkID2  {0x0f}
-	//                    1/  \1
-	//                 blkID1  blkID3
+	poll.Drop(vdr4)
+
+	require.True(poll.Finished())
+}
+
+func TestEarlyTermTraversalTransitiveTree(t *testing.T) {
+	require := require.New(t)
+
+	vdrs := bag.Of(vdr1, vdr2, vdr3, vdr4, vdr5) // k = 5
+	alphaPreference := 4
+	alphaConfidence := 4
+
+	blkID0 := ids.ID{0x00, 0x00}
+	blkID1 := ids.ID{0x0f, 0x00}
+	blkID2 := ids.ID{0xff, 0xf0}
+	blkID3 := ids.ID{0x0f, 0x0f}
+	blkID4 := ids.ID{0x00, 0x0f}
+	blkID5 := ids.ID{0x00, 0xff}
+	blkID6 := ids.ID{0xff, 0xff}
+
+	//      blk0
+	//    /     \
+	//  blk1    blk2
+	//   |       |
+	//  blk3    blk4
+	//   |       |
+	//  blk5    blk6
```

Comment:
I know this tree is not realistic in Avalanche, but for as far as what the early termination is concerned with, this is valid input.